amazoncorretto-8/mvn-entrypoint.ps1

```diff
@@ -0,0 +1,48 @@
+
+# Copy files from C:/ProgramData/Maven/Reference into ${MAVEN_CONFIG}
+# So the initial ~/.m2 is set with expected content.
+# Don't override, as this is just a reference setup
+
+function Copy-ReferenceFiles() {
```

Comment:
Yes, the mvn-entrypoint.sh is copied across the various image directories as well, this is because Docker doesn't support COPY from outside the current directory (and children) into the image. I'm sure there could be some refactoring to take that into account, but I didn't want to make that part of this changset.

Reply:
Cleaning up an existing situation is indeed something that should be deferred to a separate PR. I mentioned this here because this PR _introduces_ a lot of duplication.

Reply:
I'm definitely open to that method if that is desired. I'd also like to know if that would work for @carlossg to have that method.